webserver/views/user.py

```diff
@@ -19,7 +21,8 @@ def lastfmscraper(user_id):
         raise NotFound
     scraper = render_template(
         "user/scraper.js",
-        base_url=url_for("api_v1.submit_listen", user_id=user_id, _external=True),
+        base_url=url_for("api_v1.submit_listen",
+            user_id=user_id, _external=True),
```

Comment:
Again, this change is unnecessary. Check all other changes, I see plenty of similar ones that don't server any purpose.